src/js/components/ToggleButtonGroup/stories/Icons.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Box, ToggleButtonGroup } from 'grommet';
+import { List, MapLocation, Table } from 'grommet-icons';
+
+const optionsIcons = [
+  {
+    icon: <List />,
+    value: 'list',
+  },
+  {
+    icon: <Table />,
+    value: 'Table',
+  },
+  {
+    icon: <MapLocation />,
+    value: 'map',
+  },
+];
+
+export const UncontrolledIcons = () => (
+  <Box pad="large">
+    <ToggleButtonGroup options={optionsIcons} />
```

Comment:
I would think that this example would behave as a radio button group because I believe there should always be one option selection.